lib/split/experiment.rb

```diff
@@ -13,6 +13,13 @@ class Experiment
       :resettable => true
     }
 
+    def self.find(name)
```

Comment:
we're sure this method name doesn't collide anywhere right?

Reply:
Checked...not in the `Split::Experiment` namespace